product-service/src/functions/swagger/swagger.json

```diff
@@ -43,6 +43,39 @@
               }
             }
           }
+        },
+        "post": {
+          "tags": [
+            "product"
+          ],
+          "summary": "Create product",
+          "description": "Create product",
+          "operationId": "createProduct",
+          "produces": [
+            "application/json"
+          ],
+          "parameters": [
+            {
+              "name": "body",
+              "in": "body",
+              "description": "Product to create",
+              "required": true,
+              "schema": {
+                "$ref": "#/definitions/Product"
```

Comment:
Normally schema should be different if you are not expecting id to be passed as param

Reply:
Now I see that in order for add product enpoint to work properly id is indeed required. 
Wouldn't it be better to have that logic on a backend side of things? e.g. generate uuid when trying to create a product?

Reply:
Consumers of this API need to use uuid and not some serial numbers, so I see no problems in this approach (currently, at least).